core/src/main/java/org/apache/carbondata/core/scan/scanner/impl/BlockletFilterScanner.java

```diff
@@ -122,11 +122,11 @@ public boolean isScanRequired(DataRefNode dataBlock) {
         bitSet = ((ImplicitColumnFilterExecutor) filterExecuter)
             .isFilterValuesPresentInBlockOrBlocklet(
                 dataBlock.getColumnsMaxValue(),
-                dataBlock.getColumnsMinValue(), blockletId);
+                dataBlock.getColumnsMinValue(), blockletId, dataBlock.isMinMaxSet());
       } else {
         bitSet = this.filterExecuter
             .isScanRequired(dataBlock.getColumnsMaxValue(),
-                dataBlock.getColumnsMinValue());
+                dataBlock.getColumnsMinValue(), dataBlock.isMinMaxSet());
```

Comment:
Please use the minaxflags from the current footer and do the scan require check

Reply:
For blocklet min max comparison the footer is getting read in executor and the same min max flag information is getting used. The information serialized from driver is not getting used for blocklet min max comparison in executor